group/assignment6/parking_class.py

```diff
@@ -12,6 +12,8 @@ def __len__(self):
 
     def _compute_inverted_state(self):
         inverted_state = list(range(len(self._state)))
+        if inverted_state != sorted(self._state):
```

Comment:
maybe this check should be in helpers.py because we need the same check for 3rd nd 4th challenges which are in another class